app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for individ in people:
+        person_list.append(Person(individ["name"], individ["age"]))
+
+        if individ.get("wife") in Person.people:
+            wife = individ["wife"]
```

Comment:
don't repeat ur self, u can create a variable to avoid gettign value from dict twice here)